changelog/unreleased/pr-18367.toml

```diff
@@ -0,0 +1,4 @@
+type = "a"
```

Comment:
Let's make it a `c` because it's a change in the API error responses.

Reply:
:white_check_mark: